lib/src/base-styles/_nx-colors.scss

```diff
@@ -27,7 +27,7 @@
   --nx-color-text-stark: var(--nx-swatch-grey-05);
   --nx-color-text: var(--nx-swatch-grey-30);
   --nx-color-text-disabled: var(--nx-color-interactive-border-disabled); // match disabled text and border
-  --nx-color-text-placeholder: var(--nx-swatch-grey-60);
+  --nx-color-text-placeholder: var(--nx-swatch-grey-40);
```

Comment:
This seems like it would have a much wider affect than just this PR.  Did it not cause additional visual test diffs?

Reply:
> This seems like it would have a much wider affect than just this PR. Did it not cause additional visual test diffs?

We made this change based on [RSC-1514](https://issues.sonatype.org/browse/RSC-1514). Since this change is for light mode and `main` already has the newer snapshots, this shouldn't have any effect on the dark mode feature branch.

Reply:
> We made this change based on [RSC-1514](https://issues.sonatype.org/browse/RSC-1514). Since this change is for light mode and main already has the newer snapshots, this shouldn't have any effect on the dark mode feature branch.

Ok so it's just that only the dark mode branch has this variable for it? Has that commit on main been merged into the dark mode feature branch? Theoretically this should have been done in that merge commit not an unrelated dev branch

Reply:
> > We made this change based on [RSC-1514](https://issues.sonatype.org/browse/RSC-1514). Since this change is for light mode and main already has the newer snapshots, this shouldn't have any effect on the dark mode feature branch.
> 
> Ok so it's just that only the dark mode branch has this variable for it? Has that commit on main been merged into the dark mode feature branch? Theoretically this should have been done in that merge commit not an unrelated dev branch

Yes, only the dark mode feature branch has this variable. 

You're right. I think it'd be cleaner if I changed this in the merge commit. I'll try to push this change to the feature branch first then merge it back to this branch.

Reply:
> > > We made this change based on [RSC-1514](https://issues.sonatype.org/browse/RSC-1514). Since this change is for light mode and main already has the newer snapshots, this shouldn't have any effect on the dark mode feature branch.
> > 
> > 
> > Ok so it's just that only the dark mode branch has this variable for it? Has that commit on main been merged into the dark mode feature branch? Theoretically this should have been done in that merge commit not an unrelated dev branch
> 
> Yes, only the dark mode feature branch has this variable.
> 
> You're right. I think it'd be cleaner if I changed this in the merge commit. I'll try to push this change to the feature branch first then merge it back to this branch.

Reverted commit here: 0966c69426e0c74e49f6d4f87533acdbd635f678